config/routes.rb

```diff
@@ -817,19 +817,19 @@ def call(env)
 
           resources :application_plans, :only => [:index, :new, :create] do
             collection do
-              post :masterize
+              post :masterize # constraint to json?
```

Comment:
Is there someone we can ask to get this answer of whether or not we need the constraint to json?

Reply:
If these routes are not available in 3scale API, meaning they are used by the application only, they can be constrained. I will figure it out in time :)

Reply:
3scale API has the following endpoints for make plans default:

|||||
|-|-|-|-|
|default_admin_api_account_plan|PUT|/admin/api/account_plans/:id/default(.:format)|admin/api/account_plans#default {:format=>"xml"}|
|default_admin_api_service_service_plan|PUT|/admin/api/services/:service_id/service_plans/:id/default(.:format)|admin/api/service_plans#default {:format=>"xml"}|
|default_admin_api_service_application_plan|PUT|/admin/api/services/:service_id/application_plans/:id/default(.:format)|admin/api/application_plans#default {:format=>"xml"}|

So I think it's safe to assume `masterize` can be default and constraint do JSON.